floss/main.py

```diff
@@ -207,6 +207,98 @@ def make_parser():
 
     return parser
 
+def set_log_config(should_debug=False, should_verbose=False):
```

Comment:
This function sets logger levels, but doesn't do any other configuration. The other function (currently: `set_logging_level`) both sets logger levels and does configuration. This naming is confusing to me, and I think it should be swapped:

  - `set_log_levels`: this function
  - `set_log_config`: the other function, calls `logging.basicConfig` and also `set_log_levels`

Reply:
Done.